### PR TITLE
[WIP] Add 4.4 to changelog

### PR DESCRIPTION
**Pull request by gnestor**
Waiting on @jasongrout to review and any last-minute PRs to be included in release...

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -21,98 +21,111 @@ Upgrade to version 9+ of pip before upgrading ``notebook`` is strongly recommend
...
+.. _release-4.4.0:
+
+4.4.0
+-----
 
-- The tab icon in the browser now changes to indicate when the kernel is busy
-  (:ghpull:`1837`).
+- Allow override of output callbacks to redirect output messages
+  - Used to implement the Output widget
+  - Fixes an async bug where comm message handlers were skipped in the message processing loop
```
Comment (jasongrout) [inline docs/source/changelog.rst]: - Allow override of output callbacks to redirect output messages. This is used to implement the ipywidgets Output widget, for example.
- Fix an async bug in message handling by allowing comm message handlers to return a promise which halts message processing until the promise resolves.

**Comment by jasongrout** [comment]
Thanks @gnestor!

**Comment by gnestor** [comment]
I'm going to merge this, backport it, and run Travis on 4.x. If everything passes and no one else comments at https://github.com/jupyter/notebook/issues/2161, I will release 4.4 by EOD.

**Comment by gnestor** [comment]
@meeseeksdev backport to 4.x

**Comment by blink1073** [comment]
Probably want to hand-move the 4.x specific parts, since the PR contains some changes to 5.x changelog.

**Pull request merged**